integration/repeat_test.go

```diff
@@ -98,4 +98,15 @@ var _ = Describe("Repeat", func() {
 			Ω(session.Err).Should(gbytes.Say("--repeat and --until-it-fails are both set"))
 		})
 	})
+
+	Context("suite config overrides MustPassRepeatedly decorator", func() {
```

Comment:
I wasn't really sure where to put that one, between the following places:
+ `integration/repeat_test.go`
+ `integration/decorations_test.go`
+ `integration/flags_test.go` (where lies the test for label filter suite override)

wdyt?